backend/apps/matching/management/commands/createfakeusers.py

```diff
@@ -17,7 +17,7 @@ class Command(BaseCommand):
     # has to be "help" because we inherit from django manage.py Command, thus ignore A003
     help = (  # noqa
         "Populates the database with fake users or deletes them."
-        "Every user has their email as their password."
+        "Every user has type (A|B) as their password."
```

Comment:
@bjrne But why? I find it a lot more straightforward to just copy down the mail address.

Reply:
Copy is two clicks plus keystroke, this is one click plus keystroke ;) I you don't like it reverse it, I have several logins saved as credentials in the browser.

Reply:
It's "Helper" or "Institution" now, so the mail is easier